agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/FirstEntryPoint.java

```diff
@@ -178,6 +178,13 @@ public void onStartupSuccess() {
     String javaHome = System.getProperty("java.home");
     startupLogger.info("Java version: {}, vendor: {}, home: {}", javaVersion, javaVendor, javaHome);
 
+    boolean userHasChangedDefaultSampling =
+        configuration.sampling.percentage == null && configuration.sampling.requestsPerSecond == 5;
```

Comment:
if we log this in ConfigurationBuilder, we could avoid logging it if user explicitly sets `requestsPerSecond` to `5` (e.g. in case they want to remove the message)

Reply:
I have moved the message display in `ConfigurationBuilder`.

The other advantage is that the sampling message is now the first one displayed by Application Insights:

```
OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended
2024-05-17 17:36:52.977+02:00 INFO  c.m.a.a.i.c.ConfigurationBuilder - Some telemetry data may miss. Indeed, Application Insights applies by default from the 3.4.0 a sampling to reduce the cost. You can change the default sampling configuration: https://learn.microsoft.com/azure/azure-monitor/app/java-standalone-config
2024-05-17 17:36:54.690+02:00 INFO  c.m.applicationinsights.agent - Application Insights Java Agent 3.5.3-SNAPSHOT started successfully (PID 26512, JVM running for 2.805 s)
2024-05-17 17:36:54.691+02:00 INFO  c.m.applicationinsights.agent - Java version: 21, vendor: Oracle Corporation, 
```
The log level is info for the sampling log.